### PR TITLE
Add changelog and statistics for meta-mender/zeus-v2020.06.

### PR DESCRIPTION
**Pull request by kacf**
Signed-off-by: Kristian Amlie <kristian.amlie@northern.tech>

### CODE REVIEW

File: 202.Release-information/01.Release-notes-changelog/docs.md [hunk 1]
```diff
@@ -4,6 +4,289 @@ taxonomy:
...
+
+### Statistics
+
+A total of 7169 lines added, 4936 removed (delta 2233)
+
+| Developers with the most changesets | |
+|---|---|
+| Kristian Amlie | 79 (41.4%) |
+| Ole Petter Orhagen | 33 (17.3%) |
+| Lluis | 22 (11.5%) |
```
Comment (lluiscampos) [inline 202.Release-information/01.Release-notes-changelog/docs.md:19]: What happened to my name? It used to be "Lluis Campos". I think I had one or two commits with "Lluis" after I formatted my computer but then I fixed it...

File: 202.Release-information/01.Release-notes-changelog/docs.md [hunk 2]
```diff
@@ -4,6 +4,289 @@ taxonomy:
...
+| Drew Moseley | 19 (9.9%) |
+| pasinskim | 10 (5.2%) |
```
Comment (lluiscampos) [inline 202.Release-information/01.Release-notes-changelog/docs.md:21]: Suspicious name  also

File: 202.Release-information/01.Release-notes-changelog/docs.md (hunk 1 above)
Comment (kacf) [reply 202.Release-information/01.Release-notes-changelog/docs.md:19]: You're right, let me fix both of these.

**Comment by kacf** [comment]
Fixed! See also https://github.com/mendersoftware/integration/pull/943.

**Comment by kacf** [comment]
Cherry-picked to 2.3.x and 2.4.x as well.

**Pull request merged**